src/components/Character/renderUtils.jsx

```diff
@@ -1,19 +1,47 @@
-import React from "react";
+import React, { useContext } from "react";
 import UTILS from "SHARED/utils.js";
 import Link from "COMPONENTS/Link/";
+import { CharactersContext as Characters } from "COMPONENTS/Characters/"; 
 import { s } from "./";
 
 
 function renderButton(name, direction){
+
+	// CONTEXT
+	// ---------------------------------
+	const {
+		get: {
+			activeIndex
+		},
+		set: {
+			activeIndex: setActiveIndex
+		}
```

Comment:
Ahhh, if this is what you were referring to as to the `useState` stuff? Yeah, for this we do want to do the `useReducer` stuff then.  Basically the rule is:

- if it's only changing the state in itself then do `useState`
- if it's changing other elements as well, then do `useContext` and manage that context with `useReducer` if stuff in it needs to be changed.

This is just to keep implementations the same throughout - otherwise you gotta stop and think about which state management thing to use and it's better to just have a solid rule to follow. At least it'll be easy if it's only the active index that needs changing!